talent/onboarding/onboarding.md

```diff
@@ -0,0 +1,72 @@
+
+## Onboarding Process
+
+- This document describes the onboarding process for a new source{d} member.
+
+### Guide 
+
+- Read the source{d} [guide](https://github.com/src-d/guide) which contains all the information necessary from the moment you accept an offer. 
+
+### Hardware
+
+- A few days before you start working with us, you choose the hardware you would like to use from the [list with the hardware we provide](https://github.com/src-d/guide/blob/master/general/available_hardware.md). 
+- The first day we present it to you, so you are able to create your working environment as soon as possible.
+
+### Accounts 
+
+- From the first day you have access to source{d} Google Account, while receiving invitations to join the applications we are using:
+* Slack: the main communication channel we are using. This is the [list with our slack channels](https://github.com/src-d/guide/blob/master/communication/slack_channels.md). 
+* GitHub source{d}: you are added to our GitHub organization. The lead of your team is responsible to explain the repositories you need to be notified of and how to treat them.
+* Lever: it's a platform to support our hiring process. Everyone has access to it in order to be able to review and give feedback on candidates. 
+* Kin HR: if you want to ask days off that's the right place to do it. Also, every month you will receive the invoice of your payment through the platform. 
+* Google Calendar: here are hosted all the events. Give to our Office Manager access to your calendar in order to add the events to it. Make sure you check your calendar frequently. 
+
+- The above are the main tools we are using, but we do have a full [list with our general tools](https://github.com/src-d/guide/blob/master/general/tools.md).
+
+### Welcome Talks 
+
+- During your first week you are going to have Welcome Talks in order to get to know source{d} and every team better:
+* Welcome talks with the Founders
+* Welcome talk with the CTO
+* Welcome talk with the Product Manager
+
```

Comment:
@tsolakoua can we make a PR for a welcome talk with VP of Developer Relations, @campoy included.